protos/in_toto_attestation/predicates/human_review/vcs/v0/vcs.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package in_toto_attestation.predicates.human_review.vcs.v0;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/in-toto/attestation/go/predicates/human_review/vcs/v0";
+option java_package = "io.github.intoto.attestation.predicates.human_review.vcs.v0";
+
+message Review {
+    repeated string reviewer = 1;
```

Comment:
why is this a string? while the others are missing `Reviewer reviewer`?

Reply:
Updated to use RD based on today's discussion.